mob.go

```diff
@@ -770,6 +772,32 @@ func execute(command string, parameter []string, configuration Configuration) {
 	}
 }
 
+func clean(configuration Configuration) {
+	git("fetch", configuration.RemoteName)
+
+	currentBranch := gitCurrentBranch()
+	localBranches := gitBranches()
+
+	if currentBranch.isStale(configuration) {
+		currentBaseBranch, currentWipBranch := determineBranches(currentBranch, localBranches, configuration)
+		sayInfo("Switching to base branch " + currentBaseBranch.Name + " because wip branch " + currentWipBranch.Name + " is stale")
+		git("checkout", currentBaseBranch.Name)
+	}
+
+	for _, branch := range localBranches {
+		b := newBranch(branch)
+		if b.isStale(configuration) {
+			sayInfo("Removing stale wip branch " + b.Name)
+			git("branch", "-d", b.Name)
+		}
+	}
+
+}
+
+func (branch Branch) isStale(configuration Configuration) bool {
```

Comment:
What do you think about the function name isOrphan? I needed to Google stale. I think orphan could be a better wording because to branch has lost it’s remote origin.

Reply:
Yes, makes sense.